createdisk.sh

```diff
@@ -225,6 +226,10 @@ ${OC} --config $1/auth/kubeconfig replace -f pull-secret.yaml
 # Remove the Cluster ID with a empty string.
 ${OC} --config $1/auth/kubeconfig patch clusterversion version -p '{"spec":{"clusterID":""}}' --type merge
 
+# Add a user developer:developer with htpasswd identity provider
+${OC} --config $1/auth/kubeconfig create secret generic htpass-secret --from-literal=htpasswd=${DEVELOPER_USER_PASS} -n openshift-config
+${OC} --config $1/auth/kubeconfig apply -f htpasswd_cr.yaml
+
```

Comment:
@anjannath can you also make this user to act as sudo, to do that `oc create clusterrolebinding developer --clusterrole=sudoer --user=developer` you just need this one https://docs.openshift.com/container-platform/4.1/authentication/impersonating-system-admin.html

Reply:
Okay, doing it right away!

Reply:
@anjannath these changes you should do before we remove cluster pull secret and cluster ID because we don't know if there is any downtime for apiserver once we make those changes. should come https://github.com/code-ready/snc/blob/master/createdisk.sh#L221 here

Reply:
@praveenkumar Done!